Source Code/Ticket Management System/supportgiver_menu.c

```diff
@@ -128,17 +186,66 @@ void browseTickets()
 	}
 }
 
+void openTickets()
+{
+		wchar_t customer_email[256];
+		wchar_t title[256];
+		wchar_t type_support[256];
+		wchar_t* severity[] = {"Medium","High","Critical","Urgent" };
+		wchar_t description_support[512];
+		int severity_num,i;
+
+		system("CLS");
+		wprintf(L"Please input customer email\n");
+		wscanf(L"%s", customer_email);
+		wprintf(L"Creating a ticket\n");
+		wprintf(L"===============\n");
+		wprintf(L"Please input description\n");
+		wscanf(L"%s", description_support);
+		wprintf(L"Please input title\n");
+		wscanf(L"%s", title);
+		wprintf(L"Please input type\n");
+		wscanf(L"%s", type_support);
+		wprintf(L"Please input the option of sevrity{1.Medium,2.High,3.Critical,4.Urgent}:\n");
+		scanf("%d",&severity_num);
+		if (severity_num >= 1 && severity_num <= 4)
+		{
+			severity[severity_num];
+		}
+		else
+		{
+			printf("invaild option!!!");
```

Comment:
very professionally excited print.
Also, will crash on some systems due to not being wide.

Use `wprintf`.